src/cart/crt_init.c

```diff
@@ -220,13 +220,19 @@ prov_data_init(struct crt_prov_gdata *prov_data, crt_provider_t provider,
 
 	if (max_num_ctx > CRT_SRV_CONTEXT_NUM)
 		max_num_ctx = CRT_SRV_CONTEXT_NUM;
-
 	/* To be able to run on VMs */
 	if (max_num_ctx < CRT_SRV_CONTEXT_NUM_MIN)
 		max_num_ctx = CRT_SRV_CONTEXT_NUM_MIN;
 
 	D_DEBUG(DB_ALL, "Max number of contexts set to %d\n", max_num_ctx);
 
+	d_getenv_bool("CRT_CTX_SHARE_ADDR", &set_sep);
+	if (set_sep)
+		D_WARN("Unsupported SEP mode requested. Unset CRT_CTX_SHARE_ADDR\n");
```

Comment:
suggestion (to tackle in a follow on PR if we agree):
i'd rather not have this env variable check. are you aware anyway of anyone using that?
the following one is fine because that is more explicit in the options.

Reply:
Sure, I don't mind removing this altogether from CART. I kept it originally in case CI sets it somewhere, so that we would see it in logs and/or NLT analysis, but yeah it should be fine to remove it now.

Reply:
Addressed in https://github.com/daos-stack/daos/pull/14255